components/projects/project-card.js

```diff
@@ -0,0 +1,77 @@
+import React, { Fragment, useEffect, useState } from 'react'
+import PropTypes from 'prop-types'
+import { Stack, Box, Card, CardHeader, CardMedia, CardContent, Chip, Typography, CardActionArea, ButtonBase, useMediaQuery } from '@mui/material'
+import { Link } from '../link'
+import { Pre } from '../pre'
+import { useTheme } from '@mui/material/styles'
+import { Markdown } from '../markdown'
+import { Pattern } from './project-image'
+
+export const ProjectCard = ({project}) => {
+  const theme = useTheme();
+
+  const styles = {
+    project: {
+      margin: '1rem 0'
+    },
+    cardMedia: {
+      width: useMediaQuery(theme.breakpoints.down("md")) ? '100%' : '250px',
+      backgroundSize: 'contain',
+      backgroundPosition: 'center'
+    },
+    description: {
+      textAlign: 'left',
+      flexBasis: useMediaQuery(theme.breakpoints.down("md")) ? null:'840px',
+    },
+  }
+
+  return (
+    <Card sx={styles.project} key={project.slug} >
+      <Stack
+        direction={{ sm: 'column', md: 'row' }}
+        spacing={3}
+        flex
+      >
+        {
+          project.featuredImage.length > 0 
+          ? <CardMedia component={'img'} src={  project.featuredImage[0].url } sx={styles.cardMedia} />
+          : <Pattern />
+        }
+        <CardContent sx={styles.description}>
+          <Stack direction="row-reverse" justifyContent="space-between" >
+            {project.isActive ? (
+              <Chip label="ACTIVE" size="small" sx={{
```

Comment:
i think we'll want to have other active/inactive artifacts. it would be worth reusing this component when that time comes.